### DISCUSSION TITLE
custom arrows in fancyapp v4?

### DISCUSSION THREAD

**Discussion by steverethink** [fancyapps/ui#51] [Q&A] [answered]
whats the attribute in the template options to use your own svg's? i know closeButton is in the doc, but i don't see anything for the prev and next arrows. i would assume its something like:

```
Fancybox.bind("[data-fancybox]", {
  template: {
    prevArrow: '<svg>...</svg>';
    nextArrow: '<svg>...</svg>;
  }
});
```

**Comment by fancyapps** [comment, marked as answer]
Hi,

Excellent question! You see, the latest generation Fancybox is very unique, it is a combination of 3 components - Fancybox, Carousel and Panzoom. Each has its own plugins. This architecture has many advantages and, once you get used to it, it is very easy to use. So the navigation arrows are actually created by the carousel plugin, and it can be customized as follows:

```JS
Fancybox.bind('[data-fancybox="gallery"]', {
  Carousel: {
    Navigation: {
      prevTpl: "<<",
      nextTpl: ">>",
    },
  },
});
```

Demo - https://fancyapps.com/playground/xh

**Reply by steverethink** [reply]
ahh worked perfectly! thanks!
